src/test/java/SearchTests.java

```diff
@@ -0,0 +1,68 @@
+import POM.pages.*;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class SearchTests extends BaseTest{
+    @Test
+    public void searchASong(){
+        HomePage homePage = new LoginPage(driver).signIn("nozishka86@gmail.com","te$t$tudent");
+        BasePage basePage = new BasePage(driver).search();
+        SearchPage searchPage = new SearchPage(driver).searchForSong("Pluto");
+
+        Assert.assertTrue(searchPage.plutoFound());
+    }
+
+    @Test
+    public void searchByArtist(){
+        HomePage homePage = new LoginPage(driver).signIn("nozishka86@gmail.com","te$t$tudent");
+        BasePage basePage = new BasePage(driver).search();
+        SearchPage searchPage = new SearchPage(driver).searchByArtist();
+
+        Assert.assertTrue(searchPage.makaihFound());
+    }
+
+    @Test
+    public void searchByAlbum(){
+        HomePage homePage = new LoginPage(driver).signIn("nozishka86@gmail.com","te$t$tudent");
+        BasePage basePage = new BasePage(driver).search();
+        SearchPage searchPage = new SearchPage(driver).searchByAlbum();
+
+        Assert.assertTrue(searchPage.airbitFound());
+    }
```

Comment:
"Magic" numbers and strings are constants that have meaning beyond their immediate, literal value. 

Do not use literal values, either numeric or strings, in your code other than to define symbolic constants. Use the following pattern to define variable:

```
@Test
  public void searchNonExSong() {
    String nameSong = "Nozima";
    SearchPage searchPage = new SearchPage(driver).searchForSong(nameSong);
    Assert.assertTrue(searchPage.noneFoundDisplayed());
  }
```